backend/src/controllers/DailyWorkoutsController.js

```diff
@@ -24,7 +28,89 @@ const getDailyWorkoutsbyDate = async (req, res) => {
     }
 }
 
+
+  const insertWorkout = async (req, res) => {
+    const userId = req.user.data.user_id;
+    const { dailyworkout_id, workout_id, sets, reps } = req.body;
+    const currentDate = new Date(); // Get the current date 
+    console.log("hit");
+    try {
+        const workoutExists = await db.oneOrNone(
+            "SELECT * FROM dailyworkouts WHERE dailyworkout_id = $1",
+            [dailyworkout_id]
+        );
```

Comment:
we are using the wrong table. We should be checking if a workout exists from the `workouts` table with the `workout_id`